tools/win_compile_sanity_check.ps1

```diff
@@ -0,0 +1,28 @@
+# Script to test if a small target builds quickly on windows as expected.
+
+# Define the timeout value in seconds
+$timeout = 120
+
+# Start a timer
+$start = Get-Date
+
+# Run the command

```

Comment:
I'd suggest removing the destination directory before executing the command itself separately.